mne/io/brainvision/brainvision.py

```diff
@@ -243,44 +243,40 @@ def _read_mrk(fname, ignore_marker_types=False):
     description : array, shape (n_annots,)
         The description of each annotation.
     date_str : str
-        The recording time as a string. Defaults to empty string if no
-        recording time is found.
+        The recording time as a string. Defaults to empty string if no recording time is
+        found.
     """
     # read marker file
     with open(fname, "rb") as fid:
         txt = fid.read()
 
-    # we don't actually need to know the coding for the header line.
-    # the characters in it all belong to ASCII and are thus the
-    # same in Latin-1 and UTF-8
+    # we don't actually need to know the coding for the header line. the characters in
+    # it all belong to ASCII and are thus the same in Latin-1 and UTF-8
     header = txt.decode("ascii", "ignore").split("\n")[0].strip()
     _check_bv_version(header, "marker")
 
-    # although the markers themselves are guaranteed to be ASCII (they
-    # consist of numbers and a few reserved words), we should still
-    # decode the file properly here because other (currently unused)
-    # blocks, such as that the filename are specifying are not
-    # guaranteed to be ASCII.
+    # although the markers themselves are guaranteed to be ASCII (they consist of
+    # numbers and a few reserved words), we should still decode the file properly here
+    # because other (currently unused) blocks, such as that the filename are specifying
+    # are not guaranteed to be ASCII.
 
     try:
-        # if there is an explicit codepage set, use it
-        # we pretend like it's ascii when searching for the codepage
+        # if there is an explicit codepage set, use it we pretend like it's ASCII when
+        # searching for the codepage
```

Comment:
reflowing this comment makes punctuation necessary.

```suggestion
        # if there is an explicit codepage set, use it. We pretend like it's ASCII when
        # searching for the codepage
```